contracts/libraries/PostLib.sol

```diff
@@ -260,6 +261,30 @@ library PostLib  {
         emit ReplyCreated(userAddr, postId, parentReplyId, postContainer.info.replyCount);
     }
 
+    /// @notice Post reply
+    /// @param self The mapping containing all posts
+    /// @param userAddr Author of the reply
+    /// @param postId The post where the reply will be post
+    /// @param ipfsHash IPFS hash of document with reply information
+    /// @param messengerType The type of messenger from which the action was called
+    /// @param handle Nickname of the user who triggered the action
+    function createReplyByBot(
+        PostCollection storage self,
+        address userAddr,
+        uint256 postId,
+        bytes32 ipfsHash,
+        CommonLib.MessengerType messengerType,
+        bytes32 handle
```

Comment:
I think it makes more sense to pass handle as string to this level and convert to bytes32 here where it is actually needed as bytes32